lib/ansible/module_utils/urls.py

```diff
@@ -602,7 +602,7 @@ def get_ca_certs(self):
         to_add = False
 
         # Write the dummy ca cert if we are running on Mac OS X
-        if system == 'Darwin':
+        if system == b('Darwin'):
```

Comment:
Okay, sounds like an issue on Darwin. Let's convert to text on line 579 and then change the strings to u"" strings:
``` python

from ansible.module_utils._text import to_bytes, to_native, to_text
[...]
system = to_text(platform.system(), errors='surrogate_or_strict')
[...]
        if system == u'Linux':
[...]
        elif system == u'FreeBSD':
 [etc]
        if system == u"Darwin":
```